lightning/src/ln/msgs.rs

```diff
@@ -49,6 +51,18 @@ use crate::ln::{PaymentPreimage, PaymentHash, PaymentSecret};
 /// 21 million * 10^8 * 1000
 pub(crate) const MAX_VALUE_MSAT: u64 = 21_000_000_0000_0000_000;
 
+/// A public nonce used for Musig2, comprised of two ECPoints
```

Comment:
I don't know if the name should be labeled `MuSig2BIPPublicNonce` (ideally the bip number but no one yet to the best of my knowledge) or whatever to dissociate that's the two-point nonces variant of MuSig2 we're using (in case we introduce a variant on musig2 itself in the future for optimisations whatever)